ledger/sandbox-classic/BUILD.bazel

```diff
@@ -11,241 +11,114 @@ load("//ledger/ledger-api-test-tool:conformance.bzl", "server_conformance_test")
 load("@os_info//:os_info.bzl", "is_windows")
 load("@build_environment//:configuration.bzl", "mvn_version")
 
+sandbox_classic_deps = [
+    "//daml-lf/data",
+    "//language-support/scala/bindings",
+    "//ledger/caching",
+    "//ledger/ledger-api-auth",
+    "//ledger/ledger-api-common",
+    "//ledger/ledger-configuration",
+    "//ledger/ledger-resources",
+    "//ledger/metrics",
+    "//ledger/participant-integration-api",
+    "//ledger/sandbox-common:sandbox-common-ce",
+    "//ledger/sandbox-on-x:sandbox-classic-server",
+    "//ledger-service/cli-opts",
+    "//libs-scala/contextualized-logging",
+    "//libs-scala/ports",
+    "//libs-scala/resources",
+    "@maven//:ch_qos_logback_logback_classic",
+]
+
+# TODO sandbox-classic cleanup: De-alias
 alias(
-    name = "sandbox-classic",
-    actual = "sandbox-classic-ce",
+    name = "sandbox-classic-scala-tests-lib",
```

Comment:
Aliased this here in order to not introduce more changes to its dependencies in this PR.

Reply:
Should we wait for those changes before approving?

Reply:
No, need. Left them for a subsequent PR (to limit the scope of the changes here)